docs/source/cli_help/github.rst

```diff
@@ -57,9 +57,13 @@ Creates, checks out and manages GitHub PRs while keeping them reflected in branc
 
     --draft    Creates the new PR as a draft.
 
-``retarget-pr``:
+``retarget-pr [--branch=<branch-name>]``:
```

Comment:
Also, isn't `-b` an alias for `--branch` for the commands that already support that option? 🤔 if so, let's keep it so here as well, for consistency